src/app/components/AccountMenu/index.test.tsx

```diff
@@ -13,8 +13,20 @@ const defaultProps = {
 };
 
 const mockAccounts: Accounts = {
-  "1": { id: "1", connector: "lnd", config: "", name: "LND account" },
-  "2": { id: "2", connector: "galoy", config: "", name: "Galoy account" },
+  "1": {
```

Comment:
In another PR these could be moved to, right?:
https://github.com/getAlby/lightning-browser-extension/tree/task-nostr-account/tests/fixtures

Also these should be moved there, not sure why these neded up in `src`:  
https://github.com/getAlby/lightning-browser-extension/tree/task-nostr-account/src/fixtures